mindjunkies/forums/views.py

```
@@ -1,129 +1,189 @@
-from .models import ForumReply, ForumTopic
 
+from .models import ForumComment, ForumTopic,Reply
 
-class ForumHomeView(LoginRequiredMixin, TemplateView):
-    template_name = "forums/forum_home.html"
+
+from django.shortcuts import get_object_or_404
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic import TemplateView,CreateView
+
+from .models import Course, ForumTopic ,LikedPost
```

Ruff (0.8.2)

Redefinition of unused `Course` from line 7

(F811)